go/pkg/config/load.go

```diff
@@ -142,13 +152,15 @@ func Parse(text []byte, dir string) (conf *Config, err error) {
 func FindConfigPath(startFolder string) (configPath string, deprecatedRepositoryProjectRoot string, err error) {
 	folder := startFolder
 	for i := 0; i < maxSearchDepth; i++ {
-		configPath = filepath.Join(folder, global.ConfigFilename)
-		exists, err := files.FileExists(configPath)
-		if err != nil {
-			return "", "", fmt.Errorf("Failed to scan directory %s: %s", folder, err)
-		}
-		if exists {
-			return configPath, "", nil
+		for j := 0; j < len(global.ConfigFilenames); j++ {
```

Comment:
You can use an idiomatic Go [range loop](https://gobyexample.com/range): `for _, configFilename := range global.ConfigFilenames { [...] }`

Reply:
Done